vms/platformvm/block/builder/helpers_test.go

```diff
@@ -180,13 +185,20 @@ func newEnvironment(t *testing.T) *environment {
 		pvalidators.TestManager,
 	)
 
-	res.network = network.New(
+	res.network, err = network.New(
 		res.backend.Ctx,
 		res.blkManager,
 		res.mempool,
 		res.backend.Config.PartialSyncPrimaryNetwork,
 		res.sender,
+		time.Second,
+		100,
+		10,
+		0.1,
+		0.5,
```

Comment:
nit: I'd appreciate if we could list these values as package consts.
We have other PRs cleaning up UTs, so these consts are likely to be moved to other packages.
I could forget about this hard-coded values if they are not listed as a package-level const.
Thanks!

Reply:
Also we should probably fix the network in testEnvironments of other packages (I know, there is some code duplication).
Gonna make a PR on top of this to handle it

Reply:
Moved to constants